### ISSUE TITLE
Add numbers

### ISSUE THREAD

**Issue by stefanheule** [stefanheule/obsidian#9]
Small numbers next to the hour ticks.

**Comment by adeolemon** [comment]
Much like the "relevant only" option for the minute ticks, having only the current hour number would be pretty nice!

**Comment by adeolemon** [comment]
Mmmm, I spy a `OBSIDIAN_ONLY_RELEVANT_NUMBER`... is that related?

**Comment by stefanheule** [comment]
Yes it is.  I once tried to add numbers, but I couldn't make it look good and work out the spacing with also having the date there :/

**Comment by stefanheule** [comment]
So until I figure out the space problem, I don't know if we will see numbers any time soon unfortunately.
